packages/passport/sdk/src/Passport.ts

```diff
@@ -192,10 +192,8 @@ export class Passport {
       try {
         user = await this.authManager.getUser();
       } catch (error) {
-        if (error instanceof Error) {
-          trackError('passport', 'login', error);
-        }
         if (useCachedSession) {
+          if (error instanceof Error) trackError('passport', 'login', error);
```

Comment:
I think where the `trackError` call was before made sense, but I think the right solution is to no longer surface unknown or invalid refresh token "errors" as errors. Maybe something like this:
```
} catch (error) {
  // pseudo
  if (errorIsUnknownOrInvalidRefreshToken) {
    flow.addEvent('unknown or invalid refresh token');
  } else {
    trackError('passport', 'login', error);
  }
```

Reply:
Yeah that's correct, that way we'll know if there are any other errors being being thrown regularly